### DISCUSSION TITLE
Custom Api Base not working in Openai Handler

### DISCUSSION THREAD

**Discussion by ArnavK-09** [mindsdb/mindsdb#9010] [Get help]
Using the ` api_base ` option..

**Comment by ZoranPandovski** [comment]
Yes, I have already fixed this. For now, as a workaround, you can set the `OPENAI_API_BASE` env var.

**Reply by ArnavK-09** [reply]
I'm trying api_base opt in create engine as well as create model, it isn't working

I can't add env var
